packages/components/src/visually-hidden/style.scss

```diff
@@ -0,0 +1,13 @@
+.components-visually-hidden {
```

Comment:
This is missing:
```css
.screen-reader-text:focus {
  background-color: #eee;
  clip: auto !important;
  clip-path: none;
  color: #444;
  display: block;
  font-size: 1em;
  height: auto;
  left: 5px;
  line-height: normal;
  padding: 15px 23px 14px;
  text-decoration: none;
  top: 5px;
  width: auto;
  z-index: 100000; /* Above WP toolbar. */
}
```

See https://make.wordpress.org/accessibility/handbook/markup/the-css-class-screen-reader-text/

Reply:
oh yeh, good catch